<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>630</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Чат</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>21</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>unconnected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>313</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отключиться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adress">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>::1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nickname">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>143</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите никнейм:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nickname">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>180</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>igr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>266</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подключиться</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_My_Ip">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>488</y>
      <width>329</width>
      <height>132</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="midLineWidth">
     <number>-2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Look_At_Ip">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>451</y>
      <width>116</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Узнать мои ip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>217</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Порт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>74</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_adress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Адрес</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить сервер</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>331</width>
      <height>55</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Режим</string>
    </property>
    <widget class="QRadioButton" name="radioButton_client">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Клиент</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_server">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сервер</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_chat">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>591</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sendmes">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>420</y>
      <width>491</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendmes_spam">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>460</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Спам</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sendmes_spam">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>460</y>
      <width>491</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_emoji">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>500</y>
      <width>491</width>
      <height>72</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_JustCat">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>🐈</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Poop">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>💩</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Smile">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>😺</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Laught">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>😹</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Love">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>😻</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_WrySmile">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>😼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Cry">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>😿</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_sendmes">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_emoji">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>500</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>👈</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
